frontend-project/src/services/swagger.ts

```diff
@@ -0,0 +1,196 @@
+// This file is generated, do not manually edit this file
+// Update and use '@/scripts/generate_interface.js' instead
+
+export interface AdministrativeUnit {
+  id: string;
+  parent: string;
+  name: string;
+  category: number;
+  slug: string;
+  updatedOn: string;
+  active: boolean;
+}
+
+export interface CaseCount {
+  id: number;
+  comment: string;
+  auditedInstitutions: number[];
+  name: string;
+  responsibleUsers: number[];
+  notifiedUsers: number[];
+  featureoptions: number[];
+  tags: string[];
+  createdBy: number;
+  modifiedBy: number;
+  createdOn: string;
+  modifiedOn: string;
+  letterCount: number;
+  noteCount: number;
+  eventCount: number;
+}
+
+export interface User {
+  password: string;
+  username: string;
+  email: string;
+  firstName: string;
+  lastName: string;
+  id: number;
+}
+
+export interface Channel {
+  id: number;
+  name: string;
+  city: boolean;
+  voivodeship: boolean;
+  flatNo: boolean;
+  street: boolean;
+  postalCode: boolean;
+  houseNo: boolean;
+  email: boolean;
+  epuap: boolean;
+}
+
+export interface Collection {
+  id: number;
+  name: string;
+  comment: string;
+  public: boolean;
+  expiredOn: string;
+  query: string;
+}
+
+export interface Case {
+  id: number;
+  comment: string;
+  auditedInstitutions: number[];
+  name: string;
+  responsibleUsers: number[];
+  notifiedUsers: number[];
+  featureoptions: number[];
+  tags: string[];
+  createdBy: number;
+  modifiedBy: number;
+  createdOn: string;
+  modifiedOn: string;
+}
+
+export interface Event {
+  id: number;
+  case: number;
+  name: string;
+  date: string;
+  comment: string;
+}
+
+export interface File {
+  id: number;
+  path: string;
+  downloadUrl: string;
+  name: string;
+  letter: number;
+}
+
+export interface Letter {
+  id: number;
+  direction: string;
+  channel: number;
+  final: boolean;
+  date: string;
+  referenceNumber: string;
```

Comment:
To pole w poprzednim modelu było nieprawidłowe. Generalnie różne zawierające `_` były źle przepisane. Zatem już mamy korzyść w postaci usunięcia błędu.